services/com.mde-network.ep.toolfunctions.xtextfunction/src/main/java/com/mdenetnetwork/ep/toolfunctions/xtextfunction/RunConversionXToY.java

```diff
@@ -0,0 +1,24 @@
+package com.mdenetnetwork.ep.toolfunctions.xtextfunction;
+
+import java.io.ByteArrayOutputStream;
+
+import com.google.gson.JsonObject;
+import com.mdenetnetwork.ep.toolfunctions.core.MdeNetToolFunction;
+
+import com.mdenetnetwork.ep.toolfunctions.xtext.XtextTool;
+
+
+public class RunConversionXToY extends MdeNetToolFunction {
+
+	@Override
+	public void serviceImpl(JsonObject request, JsonObject response) throws Exception {
```

Comment:
What is the purpose of this class? It currently doesn't do anything useful...

Reply:
Agreed I'll remove it.

Reply:
Removed in 5238085fe028be75a6daea1b139452b797742ee9